src/main/resources/license_map.csv

```diff
@@ -21,6 +21,7 @@ http://opensource.org/licenses/BSD-3-Clause,BSD_3
 https://opensource.org/licenses/BSD-3-Clause,BSD_3
 EPL-2.0,EPL_2
 Eclipse Public License 2.0,EPL_2
+Eclipse Public License - v 1.0,EPL_1
```

Comment:
Please also add the url `http://www.eclipse.org/legal/epl-v10.html` and not put it in between the EPL_2's. Add it at the end of the file

Reply:
OK, I will re-create a PR for it

Reply:
OK, I will re-create a PR for it